web/app/settings/settings.screensaver.html

```diff
@@ -0,0 +1,192 @@
+<section class="screensaver-settings">
+    <div class="header">
+        <a class="btn pull-left" title="Menu" snap-toggle="left">
+            <i class="glyphicon glyphicon-menu-hamburger"></i>
+        </a>
+        <h2 translate-keep-content translate="screensaver.settings.header">Screensaver</h2>
+    </div>
+
+    <div ng-repeat="infoMessage in infoMessages" uib-alert class="alert-info" close="clearInfoMessage($index)">{{infoMessage}}</div>
+    <div ng-repeat="errorMessage in errorMessages" uib-alert class="alert-danger" close="clearErrorMessage($index)">{{errorMessage}}</div>
+
+    <form name="_form.mainForm">
+        <div ng-if="!(dashboards && dashboards.length > 0)">
+            <div class="alert alert-warning" translate-keep-content translate="screensaver.settings.error.missingdashboard">You need at least 1 dashboard to enable screensaver.</div>
+            <button class="btn btn-lg btn-primary" ng-click="cancel()" translate-keep-content translate="screensaver.settings.button.back">
+                Back
+            </button>
+        </div>
+        <div ng-if="dashboards && dashboards.length > 0">
+            <div class="scrollable box">
+                <h4 translate="{{::translations.headingGeneral}}"></h4>
+                <ul class="settings-outer">
+                    <li>
+                        <label for="ss_idleTimeout" translate-keep-content translate="screensaver.settings.idletimeout.label">Idle Timeout: </label>
+                        <div>
+                            <input id="ss_idleTimeout" class="d-block" type="number" ng-min="10" ng-change="validate()" name="idleTimeout" ng-model="config.idleTimeoutSec"
+                                required>
+                            <span>
+                                <em translate-keep-content translate="screensaver.settings.idletimeout.info">Seconds of being idle until screensaver kicks in</em>
+                            </span>
+                        </div>
+                    </li>
+
+                </ul>
+            </div>
+            <div class="scrollable box">
+                <h4 translate="{{::translations.headingonstart}}"></h4>
+                <ul class="settings-outer">
+                    <li>
+                        <label translate-keep-content translate="screensaver.settings.common.perform" for="ss_onStarType">Perform: </label>
+                        <div>
+                            <select id="ss_onStarType" class="d-block" name='onStarType' ng-model="config.onStart.type" ng-change="validate()">
+                                <option translate-keep-content translate="screensaver.settings.common.gotodashboard" value='gotodashboard'>Go to dashboard</option>
+                                <option translate-keep-content translate="screensaver.settings.onstarttype.options.slideshow" value='slideshow'>Dashboard Slideshow</option>
+                            </select>
+                            <span class='helpinfo'>
```

Comment:
Wonder whether this is really needed. Seems obvious, adds translation work and could be handled in the docs instead.